pages/add/form/detail-info-fields.js

```diff
@@ -84,6 +84,16 @@ let Tags = React.createClass({
   }
 });
 
+let IssuesLabel = React.createClass({
+  render() {
+    return(
+      <div>
+        Check any <a href="https://network.mofoprod.net/about#internet-health">Key Internet Issues</a> that relate to your project.
```

Comment:
@arush0311 looks good but I think from https://github.com/mozilla/network-pulse/issues/599#issuecomment-316149538 we want to point it to the `/issues` page on Pulse instead.